website/models.py

```diff
@@ -274,6 +275,11 @@ class Issue(models.Model):
     reporter_ip_address = models.GenericIPAddressField(null=True, blank=True)
     cve_id = models.CharField(max_length=16, null=True, blank=True)
     cve_score = models.DecimalField(max_digits=2, decimal_places=1, null=True, blank=True)
+    infringing_domain_name = models.TextField(max_length=100, null=True, blank=True)
```

Comment:
https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.Field.null

Reply:
@arkid15r Sorry for the delay. thanks for guidance